host/archival_test.go

```diff
@@ -54,7 +58,34 @@ const (
 	retryBackoffTime = 500 * time.Millisecond
 )
 
-func (s *integrationSuite) TestArchival_TimerQueueProcessor() {
+type archivalSuite struct {
+	baseIntegrationSuite
+}
+
+func TestArchivalSuite(t *testing.T) {
+	flag.Parse()
+	suite.Run(t, new(archivalSuite))
+}
+
+type durableArchivalSuite struct {
+	archivalSuite
+}
+
+func (s *durableArchivalSuite) SetupSuite() {
+	s.dynamicConfigOverrides = map[dynamicconfig.Key]interface{}{
+		dynamicconfig.DurableArchivalEnabled: dynamicconfig.BoolPropertyFn(func() bool {
+			return true
+		}),
```

Comment:
```suggestion
		dynamicconfig.DurableArchivalEnabled: true,
```
this takes a value, not a function